createdisk-library.sh

```diff
@@ -193,7 +193,11 @@ function downgrade_kernel() {
 	    ;;
     esac
 
-    ${SSH} core@${vm_ip} "sudo rpm-ostree override -C replace ${bodhi_url}"
+    ${SSH} core@${vm_ip} -- 'sudo sed -i -z s/enabled=0/enabled=1/ /etc/yum.repos.d/fedora.repo'
+    ${SSH} core@${vm_ip} -- 'sudo sed -i -z s/enabled=0/enabled=1/ /etc/yum.repos.d/fedora-updates.repo'
+    ${SSH} core@${vm_ip} "sudo rpm-ostree override replace ${bodhi_url}"
```

Comment:
for consistency can use `--` here also `${SSH} core@${vm_ip} -- "sudo .."`

Reply:
Force-pushed some changes to address this, but they are untested, need to wait for CI.

Reply:
CI is green.